README.md

```diff
@@ -128,6 +129,7 @@ module "s3_loader_enriched" {
 | <a name="input_kcl_write_min_capacity"></a> [kcl\_write\_min\_capacity](#input\_kcl\_write\_min\_capacity) | The minimum WRITE capacity for the KCL DynamoDB table | `number` | `1` | no |
 | <a name="input_max_size"></a> [max\_size](#input\_max\_size) | The maximum number of servers in this server-group | `number` | `2` | no |
 | <a name="input_min_size"></a> [min\_size](#input\_min\_size) | The minimum number of servers in this server-group | `number` | `1` | no |
+| <a name="input_purpose"></a> [purpose](#input\_purpose) | Describes the purpose which this S3 loader is being used for (RAW, ENRICHED\_EVENTS or JSON). RAW simply sinks data 1:1, ENRICHED\_EVENTS work with monitoring.statsd to report metrics (identical to RAW otherwise), SELF\_DESCRIBING partitions self-describing data (such as JSON) by its schema | `string` | `"RAW"` | no |
```

Comment:
If statsd is not enabled does ENRICHED_EVENTS fail or it just doesn't do anything different?

Reply:
I asked @benjben:

> Yep no problem you can omit statsd block, stats won't get sent to StatsD and it will still work